internal/pkg/store/sql/sqlite/cgo_free_sqllite.go

```diff
@@ -0,0 +1,85 @@
+//go:build !cgo
```

Comment:
I was trying to replace the old driver. Are there any reason to keep the cgo version?